StreetSmartAPI/StreetSmart.API/WinForms/StreetSmart.WinForms.csproj

```diff
@@ -952,6 +952,7 @@
 		</None>
 	</ItemGroup>
 	<ItemGroup Condition="'$(TargetFramework)' == 'net6.0-windows'">
+		<PackageReference Include="CefSharp.Common.NETCore" Version="123.0.60" />
```

Comment:
Is the same reference needed for .NET Framework version, or it is already working fine, without it?